react/src/components/organisms/TeaserListing/index.js

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import Collapse from '../../animations/Collapse';
+import SidebarHeading from '../../atoms/headings/SidebarHeading';
+import CompHeading from '../../atoms/headings/CompHeading';
+import Paragraph from '../../atoms/text/Paragraph';
+import Link from '../../molecules/Link';
+import GeneralTeaser from '../GeneralTeaser';
+
+import './style.css';
+
+class TeaserListing extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      open: false
+    };
+  };
+   handleClick = () => {
+    this.setState({
+      open: !this.state.open
```

Comment:
This should change so that you're using the current state that's passed into setState instead. Something like:
```
this.setState(currentState => ( { open: !currentState.open })
```

Reply:
makes sense.. Updated